www-servers/sniproxy/files/sniproxy.init

```diff
@@ -0,0 +1,29 @@
+#!/sbin/openrc-run
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+name="sniproxy daemon"
+description="Proxies incoming HTTP and TLS connections based on the hostname contained in the initial request."
+
+
+SNIPROXY_BINARY=${SNIPROXY_BINARY:-/usr/sbin/sniproxy}
```

Comment:
I still don't get the point of defining those variables if you can't adjust where the ebuild installs files anyway.